bitnami/kafka/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.13.2
+version: 12.13.3
```

Comment:
As you added a new variable, I think the version will be 12.14.0. Could you change this?

Reply:
Have Updated the version to 12.14.0

Reply:
thanks for the review comments. Have changed the version to 12.14.0